app/bundles/LeadBundle/Entity/Lead.php

```diff
@@ -377,6 +385,12 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
             ->fetchExtraLazy()
             ->build();
 
+        $builder->createOneToMany('groupScores', 'Mautic\PointBundle\Entity\GroupContactScore')
```

Comment:
It's better to use constants over strings.
```suggestion
        $builder->createOneToMany('groupScores', \Mautic\PointBundle\Entity\GroupContactScore::class)
```

Reply:
Also, I don't see this column being added in the migration.

Reply:
One more question, is there a reason for camelCase? All the other columns are using underscores as word separators.

Reply:
This is not the column name, it describes the one-to-many relationship with the `point_group_contact_score` table just like the `utmtags`. I can't see any use of snake_case when calling `createOneToMany` method.

Version20230621074925.php:64
```
$this->addSql("CREATE TABLE `{$contactScoreTableName}`
(
    `contact_id`          BIGINT UNSIGNED NOT NULL,
    `group_id`           INT UNSIGNED    NOT NULL,
    `score`               INT             NOT NULL,
    PRIMARY KEY (`contact_id`, `group_id`)

...

$this->addSql("ALTER TABLE `{$contactScoreTableName}` ADD CONSTRAINT `{$contactScoreContactFk}` FOREIGN KEY (`contact_id`) REFERENCES `{$contactTableName}` (`id`) ON DELETE CASCADE");

```